cmd/vicadmin/logs.go

```diff
@@ -298,3 +300,57 @@ func zipEntries(readers map[string]entryReader, out *zip.Writer) error {
 	}
 	return nil
 }
+
+func tailFile(wr io.Writer, file string, done *chan bool) error {
+	nlines := tailLines
+	f, err := os.Open(file)
```

Comment:
If the log file hasn't been created when this is first run then it will fail. This could easily be the case, particularly for imagec.log (in it's current form).